packages/ckeditor5-typing/src/utils/changebuffer.ts

```diff
@@ -27,61 +27,64 @@ import type { EventInfo } from '@ckeditor/ckeditor5-utils';
  *
  * To use the change buffer you need to let it know about the number of changes that were added to the batch:
  *
- *		const buffer = new ChangeBuffer( model, LIMIT );
- *
- *		// Later on in your feature:
- *		buffer.batch.insert( pos, insertedCharacters );
- *		buffer.input( insertedCharacters.length );
+ * ```ts
+ * const buffer = new ChangeBuffer( model, LIMIT );
  *
+ * // Later on in your feature:
+ * buffer.batch.insert( pos, insertedCharacters );
+ * buffer.input( insertedCharacters.length );
+ * ```
  */
 export default class ChangeBuffer {
+	/**
+	 * The model instance.
+	 */
 	public readonly model: Model;
+
+	/**
+	 * The maximum number of atomic changes which can be contained in one batch.
+	 */
 	public readonly limit: number;
 
+	/**
+	 * Whether the buffer is locked. A locked buffer cannot be reset unless it gets unlocked.
+	 *
+	 * @readonly
```

Comment:
`private` fields can be marked `readonly`. If they are not really `readonly`, marking them as such in Docs is not necessary, because they're private and code outside this class should care.